tests/integration/testmarketplace.nim

```diff
@@ -24,15 +24,15 @@ twonodessuite "Marketplace", debug1 = false, debug2 = false:
     let cid = client1.upload(data).get
     let id = client1.requestStorage(
       cid,
-      duration=10*60.u256,
+      duration=20*60.u256,
       reward=400.u256,
       proofProbability=3.u256,
-      expiry=5*60,
+      expiry=10*60,
       collateral=200.u256,
-      nodes = 5,
-      tolerance = 2).get
+      nodes = 3,
```

Comment:
Out of curiosity, is the motivation of limiting the number of nodes to hasten the completion of this test? In the past, having 5 nodes has surfaced a few issues. In saying that, I will eventually finish this PR which will have 5 or 6 nodes, so at least we'll have one test (in the future with 5/6 nodes): https://github.com/codex-storage/nim-codex/pull/976

Reply:
Yes, it now takes about 4 minutes to fill 3 slots. Something still is off that this takes so long (most likely the downloading)

Reply:
Hmm, even on the testnet I think downloading is quick... locally as an example, download takes from 12:01:02.236 to 12:01:02.346 in one of the integration tests (8 blocks)

Reply:
The proof generation might take some time. Also, I noticed on the testnet that we're doing local verification of the proofs, which is not needed and takes time on each node.

Reply:
Yes, you're right. I'm running this test with some more logging on my machine and it spends most of its time in the SaleInitialProving state.